Controller/PasswordResetController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\Controller;
+
+use SumoCoders\FrameworkMultiUserBundle\Command\PasswordReset;
+use SumoCoders\FrameworkMultiUserBundle\Command\PasswordResetHandler;
+use SumoCoders\FrameworkMultiUserBundle\Exception\InvalidPasswordConfirmationException;
+use SumoCoders\FrameworkMultiUserBundle\Form\ChangePassword;
+use SumoCoders\FrameworkMultiUserBundle\Form\ChangePasswordType;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+class PasswordResetController extends Controller
+{
+    /**
+     * @param Request $request
+     *
+     * @throws InvalidPasswordConfirmationException
+     *
+     * @return array|RedirectResponse
+     */
+    public function resetAction(Request $request)
+    {
+        $userReposioryCollection = $this->container->get('multi_user.user_repository.collection');
+        $user = $userReposioryCollection->findUserByToken($request->get('token'));
```

Comment:
ik zou zorgen dat de token in de url zit. Nu kunt ge deze url ook aanspreken zonder token. Iets als `/reset/{token}` lijkt me dan logischer, want dan wordt de token hier in je method meegegeven.